### DISCUSSION TITLE
PlusBuild Use 3DSlicer

### DISCUSSION THREAD

**Discussion by sanjuanp** [PlusToolkit/PlusLib#1076] [Q&A]
I have turned on PLUSBUILD_USE_3DSlicer option and I have set the PLUSBUILD_SLICER_BIN_DIRECTORY to my 3D Slicer binary folder, concretely where the SlicerConfig.cmake file is. It seems that it finds the Slicer package but an error appears in the line 465, where it is written:  INCLUDE( ${Slicer_USE_FILE}). The error is the following: 
```
CMake Error at C:\D\PlusB-bin\CMakeLists.txt:471 (INCLUDE): INCLUDE called with wrong number of arguments. 
include() only takes one file. 
```
Thank you in advance.

**Comment by Sunderlandkyl** [comment]
The PLUSBUILD_USE_3DSlicer option isn't actively tested/maintained.
Unless you have a specific reason to use the dependencies built by 3D Slicer it would be easier to configure Plus with the option disabled.
